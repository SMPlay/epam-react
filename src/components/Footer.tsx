import React from "react";
import { createUseStyles } from "react-jss";
import { Logo } from "./Logo";

const useStyles = createUseStyles({
  footer: {
    background: "rgb(77, 77, 77)",
  },
});

export const Footer: React.FC = () => {
  const classes = useStyles();

  return (
    <footer className={classes.footer}>
      <div className="container">
        <div className="row">
          <div className="footer-container">
            <Logo />
          </div>
        </div>
      </div>
    </footer>
  );
};
