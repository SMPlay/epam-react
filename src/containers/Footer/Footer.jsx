import React from "react";
import Wrapper from "components/Wrapper";
import Logo from "components/Logo";
import { createUseStyles } from "react-jss";

const useStyles = createUseStyles({
  footer: {
    background: "rgb(77, 77, 77)",
  },
});

const Footer = () => {
  const classes = useStyles();

  return (
    <footer className={classes.footer}>
      <Wrapper padding={20}>
        <Logo />
      </Wrapper>
    </footer>
  );
};

export default Footer;
