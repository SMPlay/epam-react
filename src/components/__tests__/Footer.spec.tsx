import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import { Footer } from "../Footer";

describe("Footer component", () => {
  it("should render", () => {
    const tree = renderer.create(
      <Router>
        <Footer />
      </Router>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
