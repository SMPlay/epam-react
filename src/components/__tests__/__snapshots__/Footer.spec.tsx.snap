// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render 1`] = `
<footer
  className="footer-0-2-1"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="footer-container"
      >
        <div>
          <a
            href="/"
            onClick={[Function]}
          >
            <h1
              className="logo-0-2-2"
            >
              netflixroulette
            </h1>
          </a>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
